__tests__/app.test.js

```diff
@@ -168,4 +167,46 @@ describe('GET /api/articles/:article_id/comments' , () => {
   expect(body.msg).toBe('Bad Request')
     })
   })
-})
\ No newline at end of file
+})
+
+describe('/api/articles/:article_id/comments.', () => {
+  test('POST 201: Responds with an array of comments from the specified article', () => {
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "butter_bridge",
+        body: "invoke yourself like a function",
+      })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: "butter_bridge",
+          body: "invoke yourself like a function",
+          article_id: 1,
+        });
+      })
+      
+    })
+    
+    test('GET 400: should give an appropriate error message when given an invalid article id', () => {
+      return request(app)
+      .get('/api/articles/notanid/comments')
+      .expect(400)
+      .then(({body}) => {
+          expect(body.msg).toBe('Bad Request')
+      })
+    })
+    test('GET 400: Responds with an bad request error message when an no comment is provided', () => {
```

Comment:
This is a good test but you could also test that the same response occurs for a missing body too.